ReactiveCocoa.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'ReactiveCocoa'
-  s.version = '4.0.2-alpha-3'
+  s.version = '4.0.3-alpha-3'
```

Comment:
Yup, it's a consequence of having to update the podspec out of lockstep with the library. The podspec maintainers have been increasing the patch release number to update a podpsec (to fix something), so this follows that convention. 

It really sucks, but because the RAC maintainers refuse to accept our help maintaining their own podspec, this is what we've got.